sphinxgallery/__init__.py

```diff
@@ -1,8 +1,8 @@
 """Sphinx Gallery
 """
 import os
-__version__ = '0.0.7'
+__version__ = '0.0.9'
```

Comment:
Yes version 0.0.8 was an emergency release. Because there was a conflict in the CSS hierarchy when merging release 0.0.7 to nilearn(There is a big difference in css if you have div.class or .class I only matters if earlier in you CSS you have defined some directives like div.class, which is the case for some sphinx themes), but I could not make a release with the same version number 0.0.7 in pypi. Thus the release to pypi was a mere local change. Then I missed to update sphinx-gallery with the 0.0.8 version number and make the release.